doc/code/qml_fourier.rst

```diff
@@ -87,10 +87,71 @@ available to a quantum model, the larger the class of functions that model can
 represent, potentially leading to greater utility for quantum machine learning
 applications.
 
+Calculating the frequencies supported by an encoding
+----------------------------------------------------
+
+For certain circuits, information on the frequency spectra :math:`\Omega_i`
+can solely be derived from the structure of the gates
+that encode the corresponding inputs :math:`x_i` (see for example `Schuld,
+Sweke and Meyer (2020) <https://arxiv.org/abs/2008.08605>`__). More precisely, if
+all input-encoding gates are of the form :math:`e^{-ix_i G}`, where :math:`G` is
+a Hamiltonian that "generates" the operation, we can deduce a maximum set of frequencies
+that can theoretically appear in :math:`\Omega_i`.  Depending on the non-input-encoding
+gates in the circuit, some of these theoretically supported frequencies may end up
+having vanishing Fourier coefficients, and :math:`\Omega_i` effectively turns out to be smaller.
+However, estimates based on the input-encoding strategy can still be useful to understand
+the potential expressivity of a type of ansatz.
+
+The theoretically supported frequencies can be computed
+using the :func:`~.pennylane.fourier.spectrum` function. To mark which gates encode
+inputs (and, for example, which ones are only used for trainable parameters), we
+have to give input-encoding gates an ``id``:
+
+.. code::
+
+    import pennylane as qml
+
+    dev = qml.device('default.qubit', wires=2)
+
+    @qml.qnode(dev)
+    def simple_circuit_marked(x):
+        qml.RX(x[0], wires=0, id="x")
+        qml.RY(x[0], wires=1, id="x")
+        qml.CNOT(wires=[1, 0])
+        return qml.expval(qml.PauliZ(0))
+
+We can then compute the frequencies supported by the input-encoding gates as:
+
+.. code::
+
+   >>> from pennylane.fourier import spectrum
+   >>> freqs = spectrum(simple_circuit_marked)([0.1])
```

Comment:
Why is it not just 
```
freqs = spectrum(simple_circuit_marked)
```
?

Reply:
The reason is simply that the qnode params could change the architecture (I think there is a test for this)

Reply:
Ah, I see :+1: